texts/rode_02.xml

```diff
@@ -1585,26 +1585,23 @@
         <div xml:id="workid59642" decls="#biblid59642">
           <head xml:id="idm140340243542736">Sne</head>
           <lg xml:id="idm140340243542352">
-            <l xml:id="idm140340243542096">Der er ingenting i Verden Saa stille som</l>
-            <l xml:id="idm140340243541648">Sne,</l>
+            <l xml:id="idm140340243542096">Der er ingenting i Verden Saa stille som Sne,</l>
```

Comment:
I see what you mean here. Obviously you are right that the second line isn't an independent line of poetry (not a semantic line of poetry). I.e., I'll accept the merge of lines, but I'll make a <lb/> instead of a new line. :^)

Reply:
The same for the other in this poem.

Reply:
OK, that makes sense. :)